tests/faker.py

```diff
@@ -0,0 +1,23 @@
+def fake_creds():
+    return {
+        'FAKE_CHANNEL': 'C12942JF92'
+    }
```

Comment:
Some other tests might need to use more credentials, and I was worried about having a global variable, hence my reason for making it a function.